src/index.ts

```diff
@@ -119,7 +119,11 @@ function realTest<Variables, Return = Variables | boolean>(
                             variables,
                             options,
                             `$eq[${index}]`
-                        ) === res
+                        ) === res ||
+                        // If the value is a string which also happens to be the name of a field in the Context,
+                        // it is possible this it is intended to be taken as a string literal, instead of the value of that field.
+                        // Example: The Scenic Route (78286bd8-b178-46dd-afbb-55685a478a71)
+                        val === res
```

Comment:
Could a better fix possibly be that we should only get the value of something from the context if it's prepended with a `$.`? I don't currently recall any accesses to Context without that (unless it's with a `$reset` op or something).

I know the SMP removes it, but it should at least keep track of it somehow.

This fix is fine with me for now, just want to spark a discussion.

Reply:
Yeah, afaik, the only instances where `$.` is omitted are in actions where you're writing to a context var, presumably because actions can only write to context vars.

Reply:
Yeah, perfectly put.

Reply:
We should probably patch out the exclusion of `$.`

Reply:
> only get the value of something from the context if it's prepended with a `$`

Changed as requested.